src/sentry/workflow_engine/endpoints/urls.py

```diff
@@ -30,6 +31,11 @@
         OrganizationDetectorDetailsEndpoint.as_view(),
         name="sentry-api-0-organization-detector-details",
     ),
+    re_path(
+        r"^(?P<organization_id_or_slug>[^/]+)/detectors/(?P<detector_id>\d+)/open-periods/$",
```

Comment:
curious about the API design, since the group and detector open period apis are doing lookups for open periods, should we create `/:org_or_slug/open-periods` as the API then have filters on `group_id` and `detector_id` for that one endpoint? 🤔